# ST_alignment_and_integration
Measurement and comparison of several slices aligning methods and data integrating mathods for spatial transcriptomics data using several ST datasets.

**Datasets:**

[1] The breast cancer dataset from Ståhl et al. is provided in this repository.

[2] The spatial transcriptomics DLPFC data are publicly available at https://github.com/LieberInstitute/spatialLIBD. The DLPFC dataset profiled by 10x Genomics Chromium platform is available at https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144136. The preprocessed DLPFC data which is used in the codes of PASTE algorism can be found https://doi.org/10.5281/zenodo.6334774. The partial_DLPFC_0.85 and partial_DLPFC_0.7 datasets are generated in the provided codes using DLPFC dataset.

[3] The mouse brain dataset: serial section 1 (sagittal–posterior; https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-1-sagittal-posterior-1-standard-1-1-0) and serial section 2 (sagittal–posterior; https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0).


**Methods:**

PASTE: https://github.com/raphael-group/paste

PASTE2: https://github.com/raphael-group/paste2

STAligner: https://github.com/zhanglabtools/STAligner

GPSA: https://github.com/andrewcharlesjones/spatial-alignment

STitch3D: https://github.com/YangLabHKUST/STitch3D

The requirements of the environment can be found in requirements.txt.
